GVFS/GVFS.UnitTests/Service/Mac/MacServiceTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿using GVFS.Common.FileSystem;
+using GVFS.Platform.Mac;
+using GVFS.Service;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Mock.FileSystem;
+using Moq;
+using NUnit.Framework;
+using System.IO;
+
+namespace GVFS.UnitTests.Service.Mac
+{
+    [TestFixture]
+    [NonParallelizable]

```

Comment:
Why is `MacServiceTests` marked as `NonParallelizable`?

Reply:
I had set this during debugging. Removed it.